src/router.js

```diff
@@ -9,10 +9,11 @@ import Runners from "./views/Runners.vue";
 import User from "./views/User.vue";
 import MergeRequests from "./views/MergeRequests.vue";
 import Projects from "./views/Projects.vue";
+import App from "./App.vue";
```

Comment:
You should use the store instead of the App component instance. 

```js
import store from "@/store";
```

And then use the state.